mobile/src/main/java/ch/epfl/sdp/mobile/ui/features/home/StatefulHome.kt

```diff
@@ -0,0 +1,66 @@
+package ch.epfl.sdp.mobile.ui.features.home
+
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.ui.Modifier
+import androidx.navigation.NavBackStackEntry
+import androidx.navigation.NavHostController
+import androidx.navigation.compose.NavHost
+import androidx.navigation.compose.composable
+import androidx.navigation.compose.currentBackStackEntryAsState
+import androidx.navigation.compose.rememberNavController
+import ch.epfl.sdp.mobile.data.api.AuthenticationApi
+import ch.epfl.sdp.mobile.ui.features.profile.StatefulProfileScreen
+import ch.epfl.sdp.mobile.ui.features.social.StatefulFollowingScreen
+
+/** The route associated to the social tab. */
+private const val SocialRoute = "social"
+
+/** The route associated to the settings tab. */
+private const val SettingsRoute = "settings"
```

Comment:
Modification : variable name should be uppercase

```suggestion
private const val SETTINGS_ROUTE = "settings"
```

Reply:
(same as above)